SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
```

Comment:
Do we need to add in README.md a link to this file?

Reply:
It seems other repos don't add such link to this security file, such as openvino, opencv.
Do we need do it?

BTW, why it reports "Token Merging" fail, I don't have right to rebuild it, could you help re-build it? @ilyachur

Reply:
I merged master branch to re-run all jobs